### DISCUSSION TITLE
Support in Backstage

### DISCUSSION THREAD

**Discussion by guilhermecgs** [copier-org/copier#1227] [Ideas]
### Actual Situation

Not supported

### Desired Situation

Render template directly within Backstage

fetch:copier_template    action

**Comment by guilhermecgs** [comment]
more info:

https://backstage.io/docs/features/software-templates/builtin-actions/

**Comment by pawamoy** [comment]
I know nothing about Backstage, but from what I understand, it's a project scaffolder? Shouldn't you open the feature request on *their* issue tracker? :thinking:  I'll move this as a discussion because any work that might happen will not happen here in Copier but in Backstage, so there's nothing actionable here for us.
